### PR TITLE
Add sep parameter for cell values, refs #495

### PR DESCRIPTION
**Pull request by mwjames**
Use cases can be found in `format-02-02-table-output-en.json`

### CODE REVIEW

File: tests/phpunit/Integration/Query/Fixtures/format-02-02-table-output-en.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
```
Comment (JeroenDeDauw) [inline tests/phpunit/Integration/Query/Fixtures/format-02-02-table-output-en.json]: We have some test code somewhere that will automatically find this file and use it?

File: tests/phpunit/Integration/Query/Fixtures/format-02-02-table-output-en.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+					"<td class=\"Has-text smwtype_txt\">bar<br />1001</td></tr>",
+					"</table>"
+				]
+			}
+		},
+		{
+			"about": "#1 table with sep",
+			"subject": "Table-with-sep-parameter",
+			"expected-output": {
+				"to-contain": [
```
Comment (JeroenDeDauw) [inline tests/phpunit/Integration/Query/Fixtures/format-02-02-table-output-en.json]: I like this naming

File: tests/phpunit/Integration/Query/Fixtures/format-02-02-table-output-en.json (hunk 1 above)
Comment (mwjames) [inline tests/phpunit/Integration/Query/Fixtures/format-02-02-table-output-en.json]: https://github.com/SemanticMediaWiki/SemanticMediaWiki/tree/master/tests#write-integration-tests-using-json-script

https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/tests/phpunit/Integration/Query/ByJsonQueryTestCaseRunnerTest.php

Comment (JeroenDeDauw) [inline tests/phpunit/Integration/Query/Fixtures/format-02-02-table-output-en.json]: Neat. Thanks for introducing this.

**Comment by kghbln** [comment]
This is finally documented for [table](https://semantic-mediawiki.org/w/index.php?title=Help%3ATable_format&diff=39188&oldid=32629) and [broadtable](https://semantic-mediawiki.org/w/index.php?title=Help%3ABroadtable_format&diff=39186&oldid=11930) Whilst I was at  it I added more fluff.

**Pull request merged**